tests/unit/task/test_diagnostic_task.py

```diff
@@ -39,20 +38,17 @@ def test_initialize_env(ext, tmp_path, monkeypatch):
     env = {}
     if ext in ('.jl', '.py'):
         env['MPLBACKEND'] = 'Agg'
+    if ext in ('.ncl', '.R'):
+        env['diag_scripts'] = str(diagnostics_path)
```

Comment:
Let's not move this around. Unless there is a reason for that that I am missing?

Reply:
ok let me move it back, shouldn't matter too much

Reply:
moved it back :+1: